openpype/hosts/webpublisher/publish_functions.py

```diff
@@ -65,7 +65,8 @@ def cli_publish(project_name, batch_path, user_email, targets):
         if isinstance(targets, str):
             targets = [targets]
         for target in targets:
-            pyblish.api.register_target(target)
+            for target_item in target.split(" "):
+                pyblish.api.register_target(target_item)
```

Comment:
```suggestion
           pyblish.api.register_target(target_item)
```

Reply:
This is necessary for triggering this from command line, as it is much more probable that one will add there `--targets "filepublish webpublish"` (which will be shown when one google it) than `--targets filepublish --targets webpublish`

Reply:
Ok, this was unnecessary as correct way (which is also in click documentation is only `--targets filepublish --targets webpublish`)